### PR TITLE
feat: add get all products endpoint

### PR DESCRIPTION
**Pull request by putragabrielll**
- make endpoint on products
- make response to endpoint products
- make controller products

### CODE REVIEW

File: src/controller/products.controller.js [hunk 1]
```diff
@@ -0,0 +1,6 @@
+exports.listAll = (req, res) => {
+  return res.json({
+    success: true,
+    message: 'Success'
```
Comment (fulsep) [inline src/controller/products.controller.js:4]: change message 'Success' to 'ok'

Comment (fulsep) [reply src/controller/products.controller.js:4]: @putragabrielll

Comment (fulsep) [inline src/controller/products.controller.js:4]: change message 'Success' to 'OK' @putragabrielll

**Review by fulsep — CHANGES_REQUESTED** [review]

File: src/controller/products.controller.js (hunk 1 above)
Comment (putragabrielll) [reply src/controller/products.controller.js:4]: Siap kak

**Pull request closed without merging**